wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -28,6 +28,8 @@
   public static final String LINK_REL_ENVIRONMENT_OUTPUT_CHANNEL = "environment/outputChannel";
   public static final String LINK_REL_ENVIRONMENT_STATUS_CHANNEL = "environment/statusChannel";
 
+  public static final String DEBUG_WORKSPACE_START = "debugWorkspaceStart";
```

Comment:
may be lowercase? Can you ensure that `case insensitive` query parameter doesn't affect implementation?

Reply:
from what I've tested, it is case sensitive. I've changed it to `debug-workspace-start`. However we're quite inconsistent with this. Latest changes seems to be lowercase with `-`, but there are many camelcase ones.

Reply:
and we have few lowercase with underscores as well. We have covered everything :D